### DISCUSSION TITLE
Organization Administrator

### DISCUSSION THREAD

**Discussion by LevioTodd** [keycloak/keycloak#34005] [Q&A]
We're building a 3rd party application that will be available online for customers. We're wanting to use Keycloak for user authentication and management. The Organizations feature seems to be correct to manage the multi-tenancy we're looking to provide.

I have most of this working using v26. However I need to be able to create a permission set for an "Organization Administrator" so that a specified user (or users) within an organization can manage their users and only their users.

Is this possible?

**Comment by pedroigor** [comment]
We are not yet supporting `Administrative Authorization` as per the feature roadmap. See https://github.com/keycloak/keycloak/issues/30180.

Right now, we are leveraging the Admin RBAC model and the `view|manage` realm roles.

But yeah, we need to support that mainly because of the Self-Service management interface for organizations. I think this is the capability you are looking for. If that is the case, are you interested in helping to contribute to this capability?

**Reply by LevioTodd** [reply]
I appreciate the reply. I'm going to have a look at the ADMIN_FINE_GRAINED_AUTHZ feature and see if it will help us get closer to what we're looking to implement. 

As per contributing there may be some options that I have. I'll advise if this is feasible.

**Reply by LevioTodd** [reply]
Actually, I would like to see if this is possible for us to contribute. Can you advise what I need to do to become a collaborator?

**Reply by pedroigor** [reply]
@LevioTodd Yeah, pretty much what @ssilvert mentioned. If you need any help, please reach out. 

@sguilhen ^

**Reply by LevioTodd** [reply]
Thanks everyone!

**Reply by MartaGolabek** [reply]
@LevioTodd did you manage to implement "Organization Administrator" feature?

**Reply by LevioTodd** [reply]
I had a look, but I was moved off to another project so I didn't get time to work on it.

**Reply by aviallon** [reply]
@ssilvert am I correct to assume that this is still unsupported?

**Reply by ssilvert** [reply]
> @ssilvert am I correct to assume that this is still unsupported?

@aviallon What do you mean by "this"?

**Reply by aviallon** [reply]
@ssilvert 
> We are not yet supporting Administrative Authorization as per the feature roadmap. See #30180.

**Reply by ssilvert** [reply]
@aviallon That is correct.
